### PR TITLE
Fix wa hospitalizations

### PR DESCRIPTION
**Pull request by ghop02**
Scrapes the hospitalization numbers from [WA dashboard](https://coronavirus.wa.gov/what-you-need-know/covid-19-risk-assessment-dashboard).

I think it's useful as a start to understand how to scrape Power BI dashboards, but its not pretty :)

### CODE REVIEW

File: src/cmdc_tools/datasets/official/WA/__init__.py [hunk 1]
```diff
@@ -50,13 +56,58 @@
...
+class PowerBIQueryDetails:
+    """IDs needed to query latest version of Power BI objects."""
+    resource_key: str
+    model_id: int
+    database_id: str
+    report_id: str
+
+
+def _get_resource_key(wa_dashboard_url):
+    # The resource key is base64 encoded in the iframe url for the dashboard.
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/WA/__init__.py:82]: how did you find this?!?!

:bow:

File: src/cmdc_tools/datasets/official/WA/__init__.py [hunk 2]
```diff
@@ -50,13 +56,58 @@
...
+@dataclasses.dataclass
+class PowerBIQueryDetails:
+    """IDs needed to query latest version of Power BI objects."""
+    resource_key: str
+    model_id: int
+    database_id: str
+    report_id: str
+
+
+def _get_resource_key(wa_dashboard_url):
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/WA/__init__.py]: Let's add a `:str` type here

We (me and @cc7768 )  haven't been great examples of this, but are currently going through a revolution of sorts to add this everywhere to make it easier for maintenance and onboarding new team members.

File: src/cmdc_tools/datasets/official/WA/__init__.py [hunk 3]
```diff
@@ -50,13 +56,58 @@
...
+
+    return PowerBIQueryDetails(
+        resource_key=resource_key,
+        model_id=model["id"],
+        database_id=model["dbName"],
+        report_id=data["exploration"]["report"]["objectId"],
+    )
+
+
+def run_query_for_county(query_details: PowerBIQueryDetails, county):
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/WA/__init__.py]: Also add a `:str` for county arg

File: src/cmdc_tools/datasets/official/WA/__init__.py [hunk 4]
```diff
@@ -171,11 +222,14 @@ def _unwrap_results(results):
 
 
 def main(counties=None):
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/WA/__init__.py]: Let's type this arg also

**Review by sglyon — CHANGES_REQUESTED** [review]
Thanks @ghop02 

No idea how you found that base64 encoded in iframe url1!!

File: src/cmdc_tools/datasets/official/WA/__init__.py (hunk 1 above)
Comment (ghop02) [reply src/cmdc_tools/datasets/official/WA/__init__.py:82]: haha I ended up looking through some of the javascript files that triggered the request and it was a bit more clear :)

**Pull request merged**